### DISCUSSION TITLE
Question: Will the files in the bucket be transferred to the new resource?

### DISCUSSION THREAD

**Discussion by A1ex-korotk0v** [noobaa/noobaa-core#8409] [General]
Hi, I have the following situation:
noobaa is deployed in k8s, two resources (minio s3) are connected to it.

There is one bucket that uses both resources in a distributed manner.
I want to connect a new minio s3 node and switch this bucket to a new resource.

Question: will the old files be transferred to the new resource and will they be available for download?

**Comment by nimrod-becker** [comment]
Not a Namespace Bucket correct?

Once you add the new resource and remove the old one (from configuration only!) a rebuild process will start.
Once the rebuild is done the old resource would be "free" of the data written by this bucket (verify its not a part of other buckets as well)
